### DISCUSSION TITLE
Customize label style

### DISCUSSION THREAD

**Discussion by somya-wittybrains** [srivastavaanurag79/react-native-paper-select#26] [Q&A]
Hi 
I have implemented react-native-paper-select for my app. I want to know what props can be used to customize the Label like font style, fontSize, or fontWeight of the Label text.

**Comment by srivastavaanurag79** [comment]
Please go through issue #9

**Reply by somya-wittybrains** [reply]
But this issue only provides a solution to apply colors to placeholder/label, I want to increase the label font size and font weight.

**Reply by srivastavaanurag79** [reply]
i don't think react native paper allows us to specifically change label font-size/font-weight
